### PR TITLE
League manager ar

### PR DESCRIPTION
**Pull request by garussell**
This PR is a result of Sunday's work.  

Issues: 
The PR adds methods in League class that seem to have the correct logic and work for our fixtures but do not produce expect results from the spec_harness when running through the actual data.  We can fix it with fresh eyes after some rest I'm sure.

Things that I believe would improve our SimpleCov coverage:
1. Include testing for edge cases
2. Make sure we are testing our conditional statements 
3. Adding additional expect statements where appropriate.
4. Use mocks/stubs for the `create_managers' and `create_leagues` methods (or find effective way to test these) since they currently do not have tests.

### CODE REVIEW

**Review by jpasko1112 — APPROVED** [review]
Thank you for the hard work on this!

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -109,31 +109,51 @@ def average_scores_by_team
...
+  end
+
+  def most_losses_team_id
+    @league_manager.most_losses_team_id
   end
 
-  # helper methods
   def find_team_id_with_name
-    @league_manager.find_team_name
+    @league_manager.find_team_id_with_name
```
Comment (danieljsauer) [inline lib/stat_tracker.rb:157]: I like this. Specific.

**Review by danieljsauer — APPROVED** [review]
Legend. Excellent work!
